lte/protos/session_manager.proto

```diff
@@ -37,6 +37,8 @@ message RuleRecord {
   uint64 rule_version = 9;
   // Uplink TEID
   uint32 teid = 10;
+  //Indicates that the report is final and the flow will be deleted
+  bool final_record = 11;
```

Comment:
Rebase the pr to latest master, this shoudn't be here.

git pull upstream master --rebase